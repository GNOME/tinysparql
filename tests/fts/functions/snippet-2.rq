SELECT ?u fts:snippet(?u, '>>>', '<<<') { ?u fts:match 'ban' } order by desc fts:rank(?u)
