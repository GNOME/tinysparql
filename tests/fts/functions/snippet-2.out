"http://www.example.org/test#1"	">>>bananas<<< lemons lemons"
"http://www.example.org/test#2"	">>>bananas<<< >>>bandanas<<< lemons"
"http://www.example.org/test#3"	">>>bananas<<<"
"http://www.example.org/test#4"	">>>bananas<<< lemons"
